libp2p/NodeTable.h

```diff
@@ -153,7 +153,7 @@ class NodeTable : UDPSocketEvents
     /// Called by implementation which provided handler to process NodeEntryAdded/NodeEntryDropped events. Events are coalesced by type whereby old events are ignored.
     void processEvents();
 
-    /// Add node. Node will be pinged.
+    /// Add node to the list of all nodes and if the node is known, also add it to the node table.
```

Comment:
Well it also pings the unknown one, maybe we should keep it in the comment.

Reply:
Good point - I'll rephrase the comment to call this out